scripts/run_experiment.py

```diff
@@ -113,15 +113,25 @@ def run_experiment(argv):
 
     args = parser.parse_args(argv[1:])
 
-    assert (os.environ.get("JOBLIB_START_METHOD", None) == "forkserver")
     if args.seed is not None:
         set_seed(args.seed)
 
+    sigint_hdlr = signal.getsignal(signal.SIGINT)
```

Comment:
Please include a large block comment explaining why this is necessary and how it works.

It seems to me that it might be more appropriate to implement this inside stateful_pool if possible.